### PR TITLE
Release notes for 1.22

### LINKED ISSUE #3140: Release notes for 1.22
### Release Notes:

* Brave Today #2863
* Rewards Changes #3009
* Update Yandex SE #3022
* Crash when trying to add or edit bookmark #3032
* URL autocompletion doesn't suggest bookmarks anymore #3034
* Crash when set as default browser and opening link from other app #3041
* Open bookmarks screen at last visited folder #3045

### PR DESCRIPTION
**Pull request by kjozwiak**
## Summary of Changes

This pull request fixes #3140

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Test Plan:

N/A

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [ ] Issue is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Review by iccub — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## [1.22](https://github.com/brave/brave-ios/releases/tag/v1.22)
+
+ - Added Brave Today news feed on the New Tab Page. ([#2863](https://github.com/brave/brave-ios/issues/2863))
+ - Removed redundant rewards setting to address user confusion. ([#3009](https://github.com/brave/brave-ios/issues/3009))
+ - Updated default search engine to Yandex for new installations in certain regions. ([#3022](https://github.com/brave/brave-browser/issues/#3022))
```
Comment (LaurenWags) [inline CHANGELOG.md]: I think this link should be `https://github.com/brave/brave-ios/issues/3022` not `https://github.com/brave/brave-browser/issues/#3022`. Change "brave-browser" to "brave-ios" and remove the # in the URL

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+ - Fixed crash when opening links from other applications while Brave set as default browser. ([#3041](https://github.com/brave/brave-browser/issues/#3041))
```
Comment (LaurenWags) [inline CHANGELOG.md]: Update URL, change "brave-browser" to "brave-ios" and remove the # in the URL

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+ - Fixed crash when adding or editing bookmarks in certain cases. ([#3032](https://github.com/brave/brave-browser/issues/#3032))
```
Comment (LaurenWags) [inline CHANGELOG.md]: Update URL, change "brave-browser" to "brave-ios" and remove the # in the URL

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+ - Fixed last bookmark location not being saved when revisiting bookmarks. ([#3045](https://github.com/brave/brave-browser/issues/#3045))
```
Comment (LaurenWags) [inline CHANGELOG.md]: Update URL, change "brave-browser" to "brave-ios" and remove the # in the URL

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,15 @@
...
+ - Fixed URL autocomplete not suggesting saved bookmarks. ([#3034](https://github.com/brave/brave-browser/issues/#3034))
```
Comment (LaurenWags) [inline CHANGELOG.md]: Update URL, change "brave-browser" to "brave-ios" and remove the # in the URL

**Review by LaurenWags — CHANGES_REQUESTED** [review]
comments left, generally need to update URLs from "brave-browser" to "brave-ios" and remove the # symbol from URLs.

File: CHANGELOG.md (hunk 1 above)
Comment (bsclifton) [reply CHANGELOG.md]: ++ on this

File: CHANGELOG.md (hunk 2 above)
Comment (bsclifton) [reply CHANGELOG.md]: ++ on this; should be https://github.com/brave/brave-ios/issues/3041

File: CHANGELOG.md (hunk 3 above)
Comment (bsclifton) [reply CHANGELOG.md]: ++ on this; should be https://github.com/brave/brave-ios/issues/3032

File: CHANGELOG.md (hunk 4 above)
Comment (bsclifton) [reply CHANGELOG.md]: ++ on this; should be https://github.com/brave/brave-ios/issues/3045

File: CHANGELOG.md (hunk 5 above)
Comment (bsclifton) [reply CHANGELOG.md]: ++ on this; should be https://github.com/brave/brave-ios/issues/3034

**Review by bsclifton — CHANGES_REQUESTED** [review]
see comments

**Comment by bsclifton** [comment]
@LaurenWags updated - ready for re-review 😄

**Review by bsclifton — APPROVED** [review]

**Review by LaurenWags — APPROVED** [review]
changes lgtm 👍

**Pull request merged**